integration_test/workflow_tests/file1.hdf

```diff
@@ -0,0 +1 @@
+shark
```

Comment:
You can probably just have one file, and upload it under two different names. Recommend removing file2.tif.

Reply:
Makes sense. Updated.